streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/kstream/KStreamTest.scala

```diff
@@ -222,42 +221,6 @@ class KStreamTest extends TestDriver {
     testDriver.close()
   }
 
-  @nowarn
-  @Test
-  def testTransformCorrectlyRecords(): Unit = {
```

Comment:
Same question as above. Seems we are lacking a similar test for `#process()` ?

Reply:
Hm... I have replace Transformer with ProcessorSupplier and transform with process. Have I missed something else?